RecoLocalTracker/SiStripClusterizer/test/step2_RAW2DIGI_L1Reco_RECO_ApproxClusters.py

```diff
@@ -0,0 +1,154 @@
+# using: 
+# Revision: 1.19 
+# Source: /local/reps/CMSSW/CMSSW/Configuration/Applications/python/ConfigBuilder.py,v 
+# with command line options: step2 --conditions 103X_dataRun2_Prompt_v2 -s RAW2DIGI,L1Reco,RECO --process reRECO -n 30 --data --era Run2_2018_pp_on_AA --eventcontent AOD --runUnscheduled --scenario pp --datatier AOD --repacked --filein file:out.root --fileout file:step2.root --no_exec
+import FWCore.ParameterSet.Config as cms
+
+from Configuration.StandardSequences.Eras import eras
+
+process = cms.Process('reRECO',eras.Run2_2018_pp_on_AA)
+
+# import of standard configurations
+process.load('Configuration.StandardSequences.Services_cff')
+process.load('SimGeneral.HepPDTESSource.pythiapdt_cfi')
+process.load('FWCore.MessageService.MessageLogger_cfi')
+process.load('Configuration.EventContent.EventContent_cff')
+process.load('Configuration.StandardSequences.GeometryRecoDB_cff')
+process.load('Configuration.StandardSequences.MagneticField_AutoFromDBCurrent_cff')
+process.load('Configuration.StandardSequences.RawToDigi_DataMapper_cff')
+process.load('Configuration.StandardSequences.L1Reco_cff')
+process.load('Configuration.StandardSequences.Reconstruction_Data_cff')
+process.load('Configuration.StandardSequences.EndOfProcess_cff')
+process.load('Configuration.StandardSequences.FrontierConditions_GlobalTag_cff')
+
+process.maxEvents = cms.untracked.PSet(
+    input = cms.untracked.int32(10)
+)
+
+# Input source
+process.source = cms.Source("PoolSource",
+    fileNames = cms.untracked.vstring('file:/eos/cms/store/hidata/HIRun2018A/HIMinimumBias5/RAW/v1/000/326/573/00000/C4CDB29C-62C9-AF4D-B2A3-5D9793BE3832.root'),
+    secondaryFileNames = cms.untracked.vstring()
+)
+
+process.options = cms.untracked.PSet(
+
+)
+
+# Production Info
+process.configurationMetadata = cms.untracked.PSet(
+    annotation = cms.untracked.string('step2 nevts:30'),
+    name = cms.untracked.string('Applications'),
+    version = cms.untracked.string('$Revision: 1.19 $')
+)
+
+# Output definition
+
+process.outputClusters = cms.OutputModule("PoolOutputModule",
+    compressionAlgorithm = cms.untracked.string('ZLIB'),
+    compressionLevel = cms.untracked.int32(7),
+    eventAutoFlushCompressedSize = cms.untracked.int32(31457280),
+    fileName = cms.untracked.string('file:step2clusters.root'),
+    outputCommands = cms.untracked.vstring(
+    'drop *',   
+	'keep *_*siStripClusters*_*_*'   
+    )
+)
+
+process.outputCompressed = cms.OutputModule("PoolOutputModule",
+    compressionAlgorithm = cms.untracked.string('ZLIB'),
+    compressionLevel = cms.untracked.int32(7),
+    eventAutoFlushCompressedSize = cms.untracked.int32(31457280),
+    fileName = cms.untracked.string('file:step2approximated.root'),
+    outputCommands = cms.untracked.vstring(
+    'drop *',   
+    'keep *_*SiStripClusters2ApproxClustersv1*_*_*',
+	'keep *_*SiStripClusters2ApproxClustersv2*_*_*', 
+    'keep *_*SiStripClusters2ApproxClustersv3*_*_*'         
```

Comment:
this look outdated?

Reply:
I wanted to keep the various version for future use but sure. Removing the list from the configuration. Fixed.